example.cpp

```diff
@@ -392,21 +385,11 @@ TEST_CASE_METHOD(CoherentUnitTestFixture, "test_decohere")
         }
     }
     std::cout << std::endl;
-    std::cout << "Most likely zero bit set outcome: ";
-    for (i = 0; i < 21; i++) {
-        if (1 << i & greatestZeroProbIndex) {
-            std::cout << "1";
-        } else {
-            std::cout << "0";
-        }
-    }
-    std::cout << std::endl;
     std::cout << "Bit probabilities:" << std::endl;
     for (i = 0; i < 21; i++) {
         std::cout << "Bit " << i << ", Chance of 1:" << qftReg->Prob(i) << std::endl;
     }
-
-}*/
+}
```

Comment:
Add some actual test validations using `REQUIRE_THAT` or some other variation, so that the test will fail if the implementation becomes defective.